src/LinkDotNet.Blog.Web/Features/AboutMe/Components/Skill/SkillTable.razor

```diff
@@ -5,40 +5,44 @@
         @if (ShowAdminActions)
         {
             <button type="button" class="btn btn-primary" @onclick="() => AddSkillDialog.Open()">
-                <i class="plus"></i>Add skill</button>
+                <i class="plus"></i> Add skill</button>
             <AddSkillDialog @ref="AddSkillDialog" SkillAdded="@AddSkill"></AddSkillDialog>
         }
     </div>
-<div class="table-container">
-    <table class="skill-table" id="skill-table">
+<div class="table-responsive">
+	<table class="table w-100 mt-3 align-top" id="skill-table">
+		<thead>
+		<tr>
+			<th class="w-10">Capability</th>
+			<th class="w-30">Familiar with</th>
+			<th class="w-30">Proficient</th>
+			<th class="w-30">Expert</th>
+		</tr>
+		</thead>
         <tbody>
-        <tr>
-            <th>Capability</th>
-            <th>Familiar with</th>
-            <th>Proficient</th>
-            <th>Expert</th>
-        </tr>
         @foreach (var skillCapabilityGroup in skills.GroupBy(s => s.Capability))
         {
             <tr ondragover="event.preventDefault();">
                 <td>@skillCapabilityGroup.Key</td>
                 @foreach (var skillLevel in ProficiencyLevel.All)
                 {
-                    <td @ondrop="@(() => HandleDrop(skillLevel))" class="proficiency-level">
+	                <td @ondrop="@(() => HandleDrop(skillLevel))" class="border-top align-top" style="min-width: 100px;">
                         @foreach (var skill in skillCapabilityGroup.Where(s => s.ProficiencyLevel == skillLevel))
                         {
-                            @if (ShowAdminActions)
+                            if (ShowAdminActions)
                             {
-                                <div draggable="true" @ondrag="@(() => currentDragItem = skill)" style="cursor: grab"
-                                 class="skill-tag">
```

Comment:
For example this is why some of the tests are failing.
There is a test looking for `.skill-tag` inside tests. 
So we have two options:
 1. Add again `.skill-tag` here without the backing razor.css (kind of just as a marker)
 2. Or replace `skill-tag` with d-inline-block me-2 my-2 px-2 py-1 rounded bg-light`

I would go for the first approach - as it makes the tests more readable